internal/service/ipam/api_address_resource_test.go

```diff
@@ -381,7 +381,7 @@ resource "bloxone_ipam_ip_space" "test" {
 }
 resource "bloxone_ipam_subnet" "test" {
     address = "10.0.0.0"
-    cidr = 24
+    cidr = 8
```

Comment:
doesn't /8 subnet seems too large for an UT ?

Reply:
Does it matter though? Given we remove this after the execution of UTs

Reply:
Changed the cidr to 24.